src/components/domain/Header/Notification/index.js

```diff
@@ -0,0 +1,52 @@
+import { useState } from 'react'
+import styled from '@emotion/styled'
+import { Popover, Modal, Icon } from '@components'
+
+const NotificationElement = styled.div`
+  position: absolute;
+  margin-left: 10px;
+  top: 5px;
+  right: 0;
+  width: 210px;
+  height: auto;
+  background-color: #743737;
+  color: white;
+  list-style: none;
+  box-shadow: rgba(0, 0, 0, 0.19) 0px 10px 20px, rgba(0, 0, 0, 0.23) 0px 6px 6px;
+  > button {
+    margin: 7px;
+    background-color: transparent;
+    border: none;
+    color: #ffffff;
+    font-size: 16px;
+    cursor: pointer;
+  }
+`
+
+const Notification = () => {
+  const [notificationpop, setNotificationpop] = useState(false)
```

Comment:
변수명을 카멜케이스로 변경해주면 좋을 것 같아요!
관련된 아래 변수들도 변경하면 좋을 것 같습니다!!

```suggestion
  const [notificationPop, setNotificationPop] = useState(false)
```